build.info

```diff
@@ -73,6 +75,13 @@ GENERATE[include/openssl/x509_vfy.h]=include/openssl/x509_vfy.h.in
 GENERATE[include/crypto/bn_conf.h]=include/crypto/bn_conf.h.in
 GENERATE[include/crypto/dso_conf.h]=include/crypto/dso_conf.h.in
 
+DEPEND[crypto/params_idx.c \
+       include/internal/param_names.h \
+       include/openssl/core_names.h]=util/perl/OpenSSL/paramnames.pm
```

Comment:
Or, third idea. Remove this dependency.

Reply:
Absolutely not.  We lack dependencies like this all over the place which makes development more error prone.
In this case, the paramname.pm script contains the list of all OSSL_PARAM names.  Not being dependent on this makes adding a name more problematic.

Reply:
Then just try it as a workaround. I cannot see what else could cause the Windows and MacOS build breakages.

Of course real fix then will have to involve build system fixes.

Reply:
Removing the dependency addresses the build issue.  Nice catch.

Reply:
@levitte any idea how to make the dependency to work?

Reply:
Well yeahok...  but this isn't the actual *cause* of the detected problem.  Basically, this is treating the symptom, not the actual problem.

Reply:
Agreed, it the symptom.